packages/esm-patient-registration-app/src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -84,33 +84,14 @@ export const esmPatientRegistrationSchema = {
   },
   personAttributes: {
     _type: Type.Array,
-    _default: [
-      {
-        name: 'Birthplace',
-        uuid: '8d8718c2-c2cc-11de-8d13-0010c6dffd0f',
-        type: 'coded',
-        concept: null,
-      },
-    ],
+    _description:
```

Comment:
It seems to me much less user-friendly to do things this way. The convention, as I understand it, is to have a per-field description so we can display help text when the user is modifying that field. I'm also a little confused on what is captured in the "type" field?

Reply:
I don't understand what this description is describing. These fields no longer exist. It's just an array of UUIDs now.